Docs 9551 - PM Admin Topics Review - ESB (#107)
mulesoft/docs-partner-manager: modules/ROOT/pages/partner-manager-configuration-objects.adoc

= Partner Manager Configuration Objects

> A message flow in Partner manager comprises the following configuration objects that execute end-to-end B2B message processing.

Reviewer: ```suggestion
A message flow in Partner manager comprises configuration objects that execute end-to-end B2B message processing:
```

* xref:partner-manager-identifiers.adoc[Partner identifiers] +
Send and receive identifiers, such as X12 ISA and GS identifiers and AS2 identifiers
* xref:endpoints.adoc[Receive and Send endpoints] +
Receive source messages or send translated messages using a standard transport protocol
* xref:partner-manager-create-message-type.adoc[Source and target message types] +

> Settings that define the structure of the message received from source endpoints, or of the messages sent to the target endpoints after translation

Reviewer: ```suggestion
Settings that define the structures of received messages and of the translated messages sent to the back end.
```

* xref:partner-manager-maps.adoc[Maps] +

> DataWeave transformation maps that convert inbound EDI transactions to the backend application format, or convert outbound transactions from backend systems to EDI format

Reviewer: ```suggestion
DataWeave transformation maps that convert inbound EDI transactions to backend application format or transactions outbound from back end to EDI format
```
